cli/src/main.rs

```diff
@@ -211,6 +213,14 @@ enum Subcommand {
         #[clap(flatten)]
         options: BenchmarkOptions,
     },
+    Profile {
```

Comment:
Can you add a nice helptext here that shows up on help? Maybe including some details around how the sampling works and mention of the caveat that the core has to be halted in regular intervals? :)